testForProjects.java

```diff
@@ -4,6 +4,7 @@
 	public static void tester(){
 		System.out.print("Hello World!");
 		System.out.print("Kyle can you edit this document?")
+                Boolean canSee = true;
```

Comment:
Okay good pull request are working .. I'm only leaving this comment to see where it posts too lol